Assets/HoloToolkit/Utilities/Scripts/SortingLayerOverride.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using System.Collections;
+
+namespace HoloToolkit.Unity
+{
+    // Overrides the sorting layer of all renderers and child renderers
+    public class SortingLayerOverride : MonoBehaviour
+    {
+        // Grabs the last layer in the project
+        public bool UseLastLayer = false;
+
+        // Only is used if UseLastLayer is false
+        public string TargetSortingLayerName = "Default";
+
+        private Renderer[] renderers;
+
+
+        void Start()
+        {
+            renderers = GetComponentsInChildren<Renderer>();
```

Comment:
We should check if it's null first before searching, that way we can give devs the option to serialize this into the prefab/scene.